python/tvm/autotvm/tuner/tuner.py

```diff
@@ -174,6 +177,16 @@ def tune(self, n_trial, measure_option, early_stopping=None, callbacks=(), si_pr
             else:
                 logger.setLevel(old_level)
 
+        if error_ct == i:
+            _, f = tempfile.mkstemp(prefix="tvm_tuning_", suffix=".log", text=True)
```

Comment:
Better to include "error" in the file name? After all, this file doesn't contain tuning logs with erros but just error messages.

Reply:
I agree. The new name is `tvm_tuning_errors_`.